src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,64 @@
+package com.aca.homework.week4.snake;
+
+public class Snake {
```

Comment:
You should decide the purpose of this class, is it responsible to hold a board pixel  or the whole snake information.

From point of snakeSize variable, this class can be a Snake
from point of x, y and symbol it is a BoardPixel